symphony/app/fbcnms-projects/workflows/package-lock.json

```diff
@@ -0,0 +1,271 @@
+{
```

Comment:
I believe package-lock.json is from the npm package manager. Please use yarn https://yarnpkg.com/ . It has a `yarn install` command that'll output the `yarn.lock` file.

Reply:
I think you can delete `package-lock.json` and run `yarn install` to generate the correct lock file. Maybe delete `node_modules` before you do that. I'm seeing an internal eslint error because of missing lockfile lines.

Reply:
I did that, but for some reason a lot of `yarn.lock` entries are now gone. Is that OK?